components/LineChart.js

```diff
@@ -0,0 +1,235 @@
+import * as d3 from 'd3';
```

Comment:
I think this would import the whole `d3`, including things that we don't use at all, such as [d3.selection](https://github.com/d3/d3-selection).

I suggest importing individual packages and functions (i.e. `import {scaleLinear} from 'd3-scale';`).